src/ralph/models/edx/base.py

```diff
@@ -125,7 +125,7 @@ class BaseEvent(BaseModelWithConfig):
     ip: Union[IPv4Address, Literal[""]]
     agent: str
     host: str
-    referer: Union[HttpUrl, Literal[""]]
+    referer: Union[AnyHttpUrl, Literal[""]]
```

Comment:
As we are not using the `referer` field in the xAPI conversion, may be we could make the validation less strict for this field ? Just checking if it's a string  or any value ? This could simplify the tests and speed up the validation. WDYT?

Reply:
Even if we are not using it (for now), we have to be pydantic on this :wink:

Reply:
Ok) Thanks) Then just downgrading it to `AnyHttpUrl` to accept `localhost` as the referer too.